src/components/AvatarGroup/__stories__/AvatarGroup.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { useCallback, useState } from "react";
 import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
-import { createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+import { createStoryMetaSettings } from "../../../storybook/functions/createStoryMetaSettings";
```

Comment:
It's looks like that we have some of the files with camel case naming and some with snake case (this is the right name im not sure) so I am now that imports looks a bit weird